app/javascript/components/editor/TestRunSummary.tsx

```diff
@@ -1,30 +1,10 @@
 import React, { useEffect, useCallback, useRef } from 'react'
 import { TestRun, TestRunStatus } from './types'
 import { TestRunChannel } from '../../channels/testRunChannel'
-import { TestRunSummaryContent } from './TestRunSummaryContent'
 import { fetchJSON } from '../../utils/fetch-json'
+import { TestsGroupList } from './TestsGroupList'
 
-const TestRunSummaryHeader = ({ testRun }: { testRun: TestRun }) => {
-  switch (testRun.status) {
-    case TestRunStatus.FAIL:
-      return (
-        <div className="summary-status failed">
-          <span className="--dot" />1 test failure
-        </div>
-      )
-    case TestRunStatus.PASS:
-      return (
-        <div className="summary-status passed">
-          <span className="--dot" />
-          All tests passed
-        </div>
-      )
-    default:
-      return null
-  }
-}
-
-export function TestRunSummary({
+export const TestRunSummary = ({
```

Comment:
What did converting this to `const` get you? Was that necessary for TypeScript to allow the properties?

Reply:
I wasn't able to assign `TestRunSummary.Header` and `TestRunSummary.Content` if this wasn't a `const`.